network_test.go

```diff
@@ -344,7 +345,9 @@ func testNetworkMachineCNI(t *testing.T, useConfFile bool) {
 
 			assert.FileExists(t, expectedCacheDirPath, "CNI cache dir doesn't exist after vm startup")
 
-			testPing(t, vmIP, 3, 5*time.Second)
+			if runtime.GOARCH != "arm64" {
```

Comment:
worth checking for `arm` as well? 

https://pkg.go.dev/internal/goarch#GOARCH

Reply:
I have tried by printing `runtime.GOARCH` to check if the arch is ARM64

Reply:
Unlike sending Ctrl + Alt + Del, networking should work on aarch64. Do you know the root cause?

Reply:
TestPing() is giving internet gateway error such as "Destination Host Unreachable" and not able to ping multiple VM's at a time.